models/articles.model.js

```diff
@@ -39,7 +39,28 @@ exports.selectCommentsByArticleId = (article_id) => {
     })
 }
 
+exports.updateArticleVotes = (update, article_id, existingVotes) => {
+    const { inc_votes } = update;
 
+    if (typeof inc_votes !== 'number') {
+        return Promise.reject({
+            status: 400,
+            msg: `invalid input`,
+        })
+    }
+
+    const newVotesValue = existingVotes + inc_votes;
+    const queryStr = "UPDATE articles SET votes = $1 WHERE article_id = $2 RETURNING *;";
+
```

Comment:
Again no need for JavaScript addition here. Simply do votes = votes + $1 to update the current votes in SQL :)